alpine/docker-entrypoint.sh

```diff
@@ -6,6 +6,9 @@ if [ ! -e /var/lib/eg/gateway.config.yml ]; then
 
       cp /usr/local/share/.config/yarn/global/node_modules/express-gateway/bin/generators/gateway/templates/basic/config/gateway.config.yml \
          /var/lib/eg/gateway.config.yml
+
+      echo >&2 "gateway.config.yml updating to expose admin port..."
+      sed -i 's/hostname: localhost/hostname: 0.0.0.0/' /var/lib/eg/gateway.config.yml
```

Comment:
This is on purpose. By default the Admin API is not exposed; in case you need that, you need to provide your own gateway.config file and mount it as a volume.